blue_modules/notifications.js

```
@@ -16,461 +16,373 @@ const NOTIFICATIONS_NO_AND_DONT_ASK_FLAG = 'NOTIFICATIONS_NO_AND_DONT_ASK_FLAG';
+
+          // (required) Called when a remote is received or opened, or local notification is opened
+          onNotification: async notification => {
+            // since we do not know whether we:
+            // 1) received notification while app is in background (and storage is not decrypted so wallets are not loaded)
+            // 2) opening this notification right now but storage is still unencrypted
+            // 3) any of the above but the storage is decrypted, and app wallets are loaded
+            //
+            // ...we save notification in internal notifications queue thats gona be processed later (on unsuspend with decrypted storage)
+
+            const payload = Object.assign({}, notification, notification.data);
+            if (notification.data && notification.data.data) Object.assign(payload, notification.data.data);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)